tasks/git/src/main/java/com/walmartlabs/concord/plugins/git/VariablesGithubTaskParams.java

```diff
@@ -0,0 +1,39 @@
+package com.walmartlabs.concord.plugins.git;
+
+import com.walmartlabs.concord.runtime.v2.sdk.MapBackedVariables;
+import com.walmartlabs.concord.runtime.v2.sdk.Variables;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+import static com.walmartlabs.concord.plugins.git.GitHubTaskParams.*;
+
+public final class VariablesGithubTaskParams {
```

Comment:
What if we move the input parsing into GitHubTaskAction implementors?

Something like
```java
public abstract class GitHubTaskAction<T extends GitHubTaskParams> {

    public abstract Map<String, Object> execute(UUID txId, GitHubApiInfo apiInfo, T input) throws Exception;
    
    public abstract T parseInput(Variables variables) throws Exception;
}
```